src/object-store/src/results.cpp

```diff
@@ -411,7 +411,7 @@ Query Results::get_query() const
             // The TableView has no associated query so create one with no conditions that is restricted
             // to the rows in the TableView.
             m_table_view.sync_if_needed();
-            return Query(*m_table, std::make_unique<TableView>(m_table_view));
+            return Query(*m_table, (std::unique_ptr<TableViewBase>)std::make_unique<TableView>(m_table_view));
```

Comment:
You can just do `make_unique<TableViewBase>` rather than use a C-style cast, right?